libfreerdp/core/pkcs11/cert_info.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * PKCS #11 PAM Login Module
```

Comment:
But we may have a problem if we use it as external dependency because it seems this project is no more maintained. 

See : 
[https://github.com/OpenSC/pam_pkcs11](https://github.com/OpenSC/pam_pkcs11)
and 
[https://sourceforge.net/p/opensc/mailman/message/35191905/](https://sourceforge.net/p/opensc/mailman/message/35191905/)

So maybe it would be better not to link the library. But I agree also that import as much code as I did is not the best solution either. I could reduce the import strictly at the minimum needed.

What do you think @akallabeth @hardening ?